test/global_test.go

```diff
@@ -135,3 +140,22 @@ func ensureVolumePresent(matchVol v1.Volume, volumes []v1.Volume) bool {
 	}
 	return present
 }
+
+func GetEnvVarByName(envVars []v1.EnvVar, name string) (error, string) {
```

Comment:
Functions that return a value or an error should return `(value, error)` not `(error, value)`.

Reply:
Agree, this is a copy paste, wanted to limit scope in this PR instead of changing everywhere. I'd like to change this to the format (value, ok) as well.